hidden/Common/Util.swift

```diff
@@ -39,6 +39,7 @@ class Util {
             guard let windowLayer = info["kCGWindowLayer"] as? Int else { continue }
             if windowLayer > 100
                 && windowLayer < 110 {
+                print(info)
```

Comment:
Still not working, many status bar items has level in range of 100..<110. 
▿ 10 elements
  ▿ 0 : 2 elements
    - key : "kCGWindowIsOnscreen"
    - value : 1
  ▿ 1 : 2 elements
    - key : "kCGWindowAlpha"
    - value : 0
  ▿ 2 : 2 elements
    - key : "kCGWindowBounds"
    ▿ value : 4 elements
      ▿ 0 : 2 elements
        - key : X
        - value : -4040
      ▿ 1 : 2 elements
        - key : Height
        - value : 1
      ▿ 2 : 2 elements
        - key : Y
        - value : 21
      ▿ 3 : 2 elements
        - key : Width
        - value : 20
  ▿ 3 : 2 elements
    - key : "kCGWindowMemoryUsage"
    - value : 1232
  ▿ 4 : 2 elements
    - key : "kCGWindowOwnerPID"
    - value : 2598
  ▿ 5 : 2 elements
    - key : "kCGWindowOwnerName"
    - value : Cloudflare WARP
  ▿ 6 : 2 elements
    - key : "kCGWindowNumber"
    - value : 113
  ▿ 7 : 2 elements
    - key : "kCGWindowStoreType"
    - value : 1
  ▿ 8 : 2 elements
    - key : "kCGWindowLayer"
    - value : 101
  ▿ 9 : 2 elements
    - key : "kCGWindowSharingState"
    - value : 0
This is my code from other project. It using to query opened windows, you can modify it, filter unrelated windows and try to find status bar menu.
```
func openedWindows() -> [WindowInfo] {
        let relativeToWindow = CGWindowID(0)
        let ownerFilter = ["Window Server", "Control Center", "loginwindow", "Control Strip", "Notification Center", "SystemUIServer", "Dock", "AccessibilityVisualsAgent", "SystemUIServer", "Dock Extra", "DockHelper", "TextInputSwitcher", "Spotlight"]
        let infos = CGWindowListCopyWindowInfo(.optionOnScreenOnly, relativeToWindow)
        do {
            let jsonData = try JSONSerialization.data(withJSONObject: infos, options: .prettyPrinted)
            let windows = try JSONDecoder().decode([WindowInfo].self, from: jsonData)
            let filteredWindows = windows.filter { window  in
                return window.windowAlpha == 1
            }.filter { window  in
                window.frame.size.width > 44 && window.frame.size.height > 44
            }.filter { window in
                !ownerFilter.contains(window.ownerName)
            }
            return filteredWindows
        } catch {
            print(error.localizedDescription)
        }
        
        return []
    }

 struct WindowInfo: Decodable {
        struct Bound: Decodable {
            let Height: Int
            let Width: Int
            let X:Int
            let Y: Int
            
            func toRect() -> CGRect {
                CGRect(x: X, y: Y, width: Width, height: Height)
            }
        }
        
        let windowAlpha: Int?
        private let bound: Bound
        let ownerName: String
        let pid: Int
        let name: String?
        var frame: CGRect {
            bound.toRect()
        }
        let number: Int
        private enum CodingKeys : String, CodingKey {
            case windowAlpha = "kCGWindowAlpha"
            case bound = "kCGWindowBounds"
            case ownerName = "kCGWindowOwnerName"
            case pid = "kCGWindowOwnerPID"
            case number = "kCGWindowNumber"
            case name = "kCGWindowName"
        }
        
        init(from decoder: Decoder) throws {
            let container = try! decoder.container(keyedBy: CodingKeys.self)
            windowAlpha = try? container.decode(Int.self, forKey: CodingKeys.windowAlpha)
            bound = try container.decode(Bound.self, forKey: CodingKeys.bound)
            ownerName = try container.decode(String.self, forKey: CodingKeys.ownerName)
            pid = try container.decode(Int.self, forKey: CodingKeys.pid)
            number = try container.decode(Int.self, forKey: CodingKeys.number)
            name = try container.decode(String?.self, forKey: CodingKeys.name)
        }
```

Reply:
Menu bar detection feature has been removed in the latest commit.